dataikuapi/dss/ml.py

```diff
@@ -643,6 +652,351 @@ def __repr__(self):
                                                           msg=self._internal_dict["message"])
 
 
+class DSSMLAssertionsParams(object):
```

Comment:
How would you expect this string representation to be ? I did not add it because I am afraid it would be too verbose if we print every assertionParam string representation of the `perAssertion` attribute.
Should we just print something like this ?
```python
DSSMLAssertionsParams('assertions': [ DSSMLAssertionParams('Assertion1',...), DSSMLAssertionParams('Assertion2',...), DSSMLAssertionParams('Assertion3',...), ...])
```

Reply:
Yes, or even just `DSSMLAssertionsParams(... internal_dict ...)` for simplicity, I see that as a helper when building scripts in a notebook.